tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesPage.xaml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pg:ToolPage
+    x:Class="DevHome.QuietBackgroundProcesses.UI.Views.QuietBackgroundProcessesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.QuietBackgroundProcesses.UI.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:pg="using:DevHome.Common"
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
+    mc:Ignorable="d">
```

Comment:
Nit: if you're not using the xaml designer (which I am pretty sure you can't) you can remove this line, then you can also remove other ones (Visual Studio will show you)